src/util/native_fs_utils.js

```diff
@@ -712,6 +712,30 @@ function translate_error_codes(err, entity) {
     return err;
 }
 
+/**
+ * lock_and_run acquires a flock and calls the given callback after
```

Comment:
```suggestion
 * lock_and_run acquires a fcntl and calls the given callback after
```

Initially we had `flock` but later we replaced it with `fcntl` based lock and I totally forgot to update it in the comment here.